### ISSUE TITLE
Markdown build fails with DITA-OT 3.5

### ISSUE THREAD

**Issue by jason-fox** [jelovirt/org.lwdita#70]
Re-raising https://github.com/dita-ot/dita-ot/issues/3508 in appropriate queue.

## Expected Behavior

```
./bin/dita --format markdown -i ./docsrc/samples/taskbook.ditamap ../out -d
```

Builds markdown output

## Actual Behavior

Markdown Fails with an error message

## Possible Solution

I guess this is due to the removal of`fullditatopicfile` as an  [ANT parameter](https://www.dita-ot.org/dev/topics/migrating-to-3.5.html#migrating-to-3.5__3.5-legacy-target-removal).

## Steps to Reproduce

```
./bin/dita --format markdown -i ./docsrc/samples/taskbook.ditamap ../out -d
```
## Copy of the error message, log file or stack trace

```txt
dita.inner.topics.markdown.common:
Property "fullditatopicfile" has not been set
     [xslt] Using class org.apache.tools.ant.taskdefs.optional.TraXLiaison
  [antcall] Exiting /Users/jasonfox/Workspace/dita/dita-ot-3.5-Y/build.xml.
Error: Includesfile /var/folders/79/f71dhgdd4nz6hyv8p173ypb40000gn/T/temp20200430145300596/${fullditatopicfile} not found.
```

## Environment

* DITA-OT version: 3.5
* Java 11
* Mac

**Comment by infotexture** [comment]
Reproduced with clean Homebrew installation of DITA-OT 3.5:

```
Error: Includesfile /var/folders/…/T/temp…/${fullditatopicfile} not found.
```

@jelovirt Looks like the LwDITA plug-in was not updated for 3.5 compatibility prior to the 3.5 release.

**Comment by jelovirt** [comment]
Fixed in 2.4.0
